www/src/components/account/billing/BillingSubscriptionProvider.tsx

```diff
@@ -31,7 +31,7 @@ function BillingSubscriptionProvider({ children }: BillingSubscriptionProviderPr
   const isProPlan = useMemo(() => !!subscription?.plan?.id && (subscription.plan.id === proPlatformPlan?.id || subscription.plan.id === proYearlyPlatformPlan?.id), [subscription, proPlatformPlan, proYearlyPlatformPlan])
   const isEnterprisePlan = useMemo(() => !!subscription?.plan?.id && subscription.plan.id === enterprisePlatformPlan?.id, [subscription, enterprisePlatformPlan])
   const isPaidPlan = useMemo(() => isProPlan || isEnterprisePlan, [isProPlan, isEnterprisePlan])
```

Comment:
fwiw, there's actually an easier check here that involves seeing if the account has a subscription at all.  Lots of this logic if quite brittle